sdk/communication/communication-sms/review/communication-sms.api.md

```diff
@@ -7,31 +7,40 @@
 import { KeyCredential } from '@azure/core-auth';
 import { OperationOptions } from '@azure/core-http';
 import { PipelineOptions } from '@azure/core-http';
-import { RestResponse } from '@azure/core-http';
 import { TokenCredential } from '@azure/core-auth';
 
 // @public
-export interface SendOptions extends OperationOptions {
-    enableDeliveryReport?: boolean;
+export class SmsClient {
+    constructor(connectionString: string, options?: SmsClientOptions);
+    constructor(url: string, credential: KeyCredential, options?: SmsClientOptions);
+    constructor(url: string, credential: TokenCredential, options?: SmsClientOptions);
+    send(_sendRequest: SmsSendRequest, _options?: SmsSendOptions): Promise<SmsSendResult[]>;
 }
 
 // @public
-export interface SendRequest {
-    from: string;
-    message: string;
-    to: string[];
+export interface SmsClientOptions extends PipelineOptions {
 }
 
 // @public
-export class SmsClient {
-    constructor(connectionString: string, options?: SmsClientOptions);
-    constructor(url: string, credential: KeyCredential, options?: SmsClientOptions);
-    constructor(url: string, credential: TokenCredential, options?: SmsClientOptions);
-    send(sendRequest: SendRequest, options?: SendOptions): Promise<RestResponse>;
+export interface SmsSendOptions extends OperationOptions {
+    enableDeliveryReport?: boolean;
```

Comment:
This should not be nullable anymore, right?

Reply:
It should be optional here but set to false by default